frontend/src/components/pages/profile/components/posts/Posts.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import EditIcon from '@material-ui/icons/Edit';
+import CloseIcon from '@material-ui/icons/Close';
+import UserPhoto from '../../../../mocks/user.png';
+import { Link } from 'react-router-dom';
+import './posts.scss';
+
+type PostsProps = {
+  posts: {
+    "id": number,
+    "body": string,
+    "date": string,
+    "time": string,
+    "user": {
+      "id": number,
+      "firstName": string,
+      "lastName": string,
+      "photoUrl": string
+    }
+  }[]
+}
```

Comment:
please extract all types the the separate file and export them - so no duplicates (like for user here)

Use single quotes '